src/data/structures/structures_creator.py

```diff
@@ -0,0 +1,57 @@
+import string
+
+
+class StructCreator:
```

Comment:
what about renaming to `StructureCreator`?
```suggestion
class StructureCreator:
```